api/src/TestHelpers/BehatFixtures.php

```diff
@@ -137,21 +184,48 @@ private function createAdminUsers()
     }
 
     private function createDeputies()
+    {
+        $this->createLays();
+        $this->createProfs();
+    }
+
+    private function createLays()
     {
         $this->layNotStarted = $this->userTestHelper
-            ->createUser(null, User::ROLE_LAY_DEPUTY, sprintf('lay-not-started-%s@publicguardian.gov.uk', $this->testRunId));
-        $this->addClientsAndReportsToDeputy($this->layNotStarted, false, false);
+            ->createUser(null, User::ROLE_LAY_DEPUTY, sprintf('lay-not-started-%s@t.uk', $this->testRunId));
+        $this->addClientsAndReportsToLayDeputy($this->layNotStarted, false, false);
 
-        $this->layCompletedNotSubmitted = $this->userTestHelper
-            ->createUser(null, User::ROLE_LAY_DEPUTY, sprintf('lay-completed-not-submitted-%s@publicguardian.gov.uk', $this->testRunId));
-        $this->addClientsAndReportsToDeputy($this->layCompletedNotSubmitted, true, false);
+        $this->layCompleted = $this->userTestHelper
+            ->createUser(null, User::ROLE_LAY_DEPUTY, sprintf('lay-completed-%s@t.uk', $this->testRunId));
+        $this->addClientsAndReportsToLayDeputy($this->layCompleted, true, false);
 
         $this->laySubmitted = $this->userTestHelper
-            ->createUser(null, User::ROLE_LAY_DEPUTY, sprintf('lay-submitted-%s@publicguardian.gov.uk', $this->testRunId));
-        $this->addClientsAndReportsToDeputy($this->laySubmitted, true, true);
+            ->createUser(null, User::ROLE_LAY_DEPUTY, sprintf('lay-submitted-%s@t.uk', $this->testRunId));
+        $this->addClientsAndReportsToLayDeputy($this->laySubmitted, true, true);
+    }
+
+    private function createProfs()
```

Comment:
So glad you've given us this fixture setup to make use of as I would have found this challenging. Great stuff :)

Reply:
Haha I wasn't looking forward to it and it was a lot of trial and error. Like I said in the PR comments, there may be some of the setup which is quite right when we come to do more org specific pages but at least we have a starting user now.